hack/go-lint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+if [ "$IS_CONTAINER" != "" ]; then
+  golint -set_exit_status "${1:-.}"
+else
+  docker run --rm --env IS_CONTAINER='TRUE' \
+    -v "$PWD":/go/src/github.com/openshift/installer \
+    -w /go/src/github.com/openshift/installer \
+    --entrypoint sh quay.io/coreos/golang-testing \
+    ./hack/go-lint.sh installer/...
```

Comment:
I'd like to keep the script sig the same regardless of `IS_CONTAINER`.  so whatever you use in the earlier block (`${@}` or `${1:-.}`), you should use here too.